web/shapeworks/src/store/constants.ts

```diff
@@ -81,3 +81,5 @@ export const COLORS = [
     [255, 255, 153],
     [177, 89, 40],
 ];
+
+export const SAMPLES_PER_PAGE = 12;
```

Comment:
Can we add a `DEEPSSM_` prefix to this since it's going in the global constants?

Reply:
Done in [571b8d8](https://github.com/girder/shapeworks-cloud/pull/370/commits/571b8d8d804678ddb2ce77b74524f463c9633330)